app/views/super_admin/organisations/show.html.erb

```diff
@@ -44,7 +45,7 @@
         <%= render partial: "shared/manage_ips", locals: { location: location, show_ip_controls: false } %>
       <% end %>
     <% end %>
-
+    <%== pagy_nav_govuk(@pagy, link_extra: 'class: "govuk-link"') %>
```

Comment:
the `link_extra` appears to want raw html, so this should be `class="govuk-link"` - all instances.

Reply:
good catch!